server/handlers/attachments_test.go

```diff
@@ -274,7 +274,7 @@ func TestHandleAttachments(t *testing.T) {
 				p.On("GetClientForApp").Return(client).Times(1)
 				p.On("GetAPI").Return(mockAPI).Times(2)
 				p.On("GetMaxSizeForCompleteDownload").Return(1).Times(1)
-				p.On("GetMetrics").Return(mockmetrics).Times(1)
```

Comment:
This is a wider conversation, but in general these types of tests aren't terribly valuable. I'd love to start a conversation around "e2e tests" (all written in Go) that minimizes the use of mocks unless absolutely needed. We adopted this approach in Playbooks with great success.